.env.example

```diff
@@ -41,7 +41,11 @@ HCAPTCHA_SECRET=
 
 # Redis
 REDIS_PASSWORD=!ChangeThisRedisPass!
-REDIS_DNS=redis://${REDIS_PASSWORD}@redis
+REDIS_DNS=redis://${REDIS_PASSWORD}@127.0.0.1:6379
+# Or when using Redis Socket:
+#REDIS_DNS=REDIS_DNS=redis://${REDIS_PASSWORD}/var/run/redis/redis-server.sock
+# Or socket without password:
+#REDIS_DNS=REDIS_DNS=redis:///var/run/redis/redis-server.sock
```

Comment:
was it intentional the env vars were duplicated on both these likes? `REDIS_DNS=REDIS_DNS=`

Reply:
No it was not thanks for spotting.